infrastructure-as-code/aws-lambda/README.md

```diff
@@ -0,0 +1,8 @@
+# Terraforming EC2 lifecycles with AWS Lambda
+This Terraform configuration provides examples of:
+ - AWS Lambda functions for performing various ec2 cleanup tasks
+ - IAM policy required to automate Lambda functions
+ - Cloudwatch roles for scheduling Lambda runs
+ - Simple Email Service
+ - Slack integration
+## 
```

Comment:
I think this is a first cut, but at some point it'd be great to get the README in the Guide template format --> https://docs.google.com/document/d/17zk1a_azbAp0ph4_D_xh8E-LJ6tIFR_5C967ik1Rj9c/edit#heading=h.efo7z8nz6mqv

Reply:
Yep, i'll give it the full treatment.  Just getting my python brain cells firing again.  I haven't written python for nearly 5 years.

Reply:
After I made this comment it was clear this was an early PR haha, carry on